chains/chains_dev.json

```diff
@@ -355,20 +355,20 @@
             }]
         },
         "assets": [{
-                "isUtility": true,
-                "id": "887a17c7-1370-4de0-97dd-5422e294fa75",
-                "name": "polkadot",
-                "symbol": "dot",
-                "precision": 10,
-                "priceId": "polkadot",
-                "icon": "https://raw.githubusercontent.com/soramitsu/shared-features-utils/master/icons/tokens/coloured/DOT.svg",
-                "color": "FF0066",
-                "purchaseProviders": [
-                    "moonpay",
-                    "ramp"
-                ],
-                "type": "normal"
-            },
+            "isUtility": true,
+            "id": "887a17c7-1370-4de0-97dd-5422e294fa75",
+            "name": "polkadot",
+            "symbol": "dot",
+            "precision": 10,
+            "priceId": "polkadot",
+            "icon": "https://raw.githubusercontent.com/soramitsu/shared-features-utils/master/icons/tokens/coloured/DOT.svg",
+            "color": "FF0066",
+            "purchaseProviders": [
+                "moonpay",
+                "ramp"
+            ],
+            "type": "normal"
+        },
             {
                 "id": "ea33432f-9bd9-4d42-93bc-cd45a0e8a44c",
```

Comment:
Let's perform file formatting and adding changes for EVM support in separated PRs. It's too difficult to review this PR for me because too many different changes